system76driver/actions.py

```diff
@@ -1510,3 +1510,31 @@ class i915_initramfs(FileAction):
 
     def describe(self):
         return _('Add i915 driver to initramfs')
+
+class hda_force_enable_audio(FileAction):
```

Comment:
Since this is specific to an output and not Intel HDA, I would recommend to call this displayport_1_force_enable_audio or meer6_force_enable_audio

Reply:
Renamed the function to be specific to the port. (Seems better to keep it non-machine-specific so it can be reused later, although the note about the physical port would still be machine-specific.)